src/Microsoft.Android.Sdk.ILLink/SetupStep.cs

```diff
@@ -53,6 +53,7 @@ protected override void Process ()
 				InsertAfter (new GenerateProguardConfiguration (proguardPath),  "CleanStep");
 
 			InsertAfter (new StripEmbeddedLibraries (),  "CleanStep");
+			//InsertAfter (new RemoveResourceDesigner (),  "CleanStep");
```

Comment:
when we finally get this working in .net 5/6 not sure if this is the best place to insert this step.

Reply:
if we want it at the same place as in the "old" linker, move it one line up.